Makefile.dep

```diff
@@ -606,6 +606,14 @@ ifneq (,$(filter random,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter random_seed,$(USEMODULE)))
+  USEMODULE += luid
+  USEMODULE += xtimer
+
+  FEATURES_REQUIRED += periph_hwrng
+  FEATURES_REQUIRED += periph_adc
```

Comment:
Correct and I also don't really like it. But IMO there is currently no other way. We do not know which resources are available at the time we pull in dependencies. So no way to conditionally include them. Am I mistaken?